integration_tests/test_solr_search_results.py

```diff
@@ -0,0 +1,103 @@
+import pytest
+import requests
+import typing
+import os
+
+
+@pytest.fixture
+def rsession():
+    return requests.session()
+
+
+@pytest.fixture
+def solr_url():
+    solr_url = os.getenv("INPUT_SOLR_URL")
+    if solr_url is None:
+        solr_url = "https://mars.cyverse.org/thing/select"
+    return solr_url
+
+
+def headers():
+    return {
+        "accept": "application/json",
+        "User-Agent": "iSamples SOLR Integration Bot 2000",
+    }
+
+
+authority_ids = [
+    "SESAR",
+    "OPENCONTEXT",
+    "GEOME",
+    "SMITHSONIAN",
+]
+
+
+def _send_solr_query(rsession: requests.sessions, solr_url: str, query: str) -> typing.List[typing.Dict]:
+    params = {
+        "start": 0,
+        "limit": 10,
+        "fl": "*",
+        "q": f"searchText:{query}",
+    }
+    res = rsession.get(solr_url, headers=headers(), params=params)
+    response_dict = res.json()
+    docs = response_dict.get("response").get("docs")
+    return docs
+
+
+def _value_in_search_text(value: str, doc: typing.Dict) -> bool:
+    appears_in_search_text = False
+    for searchText in doc["searchText"]:
+        if value.lower() in searchText.lower():
+            appears_in_search_text = True
+            break
+    return appears_in_search_text
+
+
+@pytest.mark.parametrize("authority_id", authority_ids)
+def test_solr_query_by_source(
+    rsession: requests.sessions, solr_url: str, authority_id: str
+):
+    docs = _send_solr_query(rsession, solr_url, authority_id)
+    for doc in docs:
+        appears_in_search_text = _value_in_search_text(authority_id, doc)
+        assert appears_in_search_text
+
+
+opencontext_projects = [
+    "Çatalhöyük Zooarchaeology",
```

Comment:
@datadavev when we discussed this one this morning, I though it was failing due to some character encoding issues.  Thankfully, it turned out that wasn't the case -- there are actually two opencontext projects with roughly this title, and solr is preferring the other one.  Given that both terms (Çatalhöyük, Zooarchaeology) appear in `searchText`, I loosened the check and treated this as a pass.